setup.py

```diff
@@ -178,6 +178,8 @@ def version(self):
 extra_reqs = {
     'test': ['pytest', 'pytest-cov', 'numpy>=1.4.1', 'packaging']
 }
+if os.getenv('NUMPY') == '0':
+    extra_reqs['test'].remove('numpy>=1.4.1')
```

Comment:
No, let's remove the NUMPY env var, and remove numpy from the test requirements. This means that on Travis we'll need to install `shapely[all]`.